### ISSUE TITLE
emb-m-10 - Tabs

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#27]
emb-m-10	Tabs	Yes	2-High	

Tabs with "non-compliance" indicators are not announced as such to screen reader users.

Ensure that in addition to the visual "!" indicator, a text equivalent is provided for screen reader users, e.g. "contains non-compliant entries".
